static/styles/portico/portico_signin.css

```diff
@@ -868,7 +874,9 @@ button#register_auth_button_gitlab {
         }
     }
 
-    #id_email {
+    #id_email,
+    #support_from,
+    #support_realm {
```

Comment:
I'm not sure where this CSS should go, but I think we want a `font-size: 1em` rather than the "default" 1.2 used for login pages and the like, or to make the input boxes wider.  Not sure which is better, but it feels like there isn't space for much text, which seems wrong.